packages/core/src/__test__/selector.test.ts

```diff
@@ -0,0 +1,66 @@
+import { createSelector } from '../selector';
+import { memoize, shallowComparer } from '../utils';
+
+jest.mock('../utils');
+
+const memoizeMock = jest.mocked(memoize);
+memoizeMock.mockReturnValue(jest.fn);
+
+describe('core', () => {
+  describe('selector', () => {
+    describe('createSelector', () => {
+      it('passes arguments to memoize', () => {
+        const func = jest.fn();
+        const comparer = jest.fn();
+
+        createSelector(func, jest.fn(), comparer);
+
+        expect(memoizeMock).toBeCalledTimes(1);
+        expect(memoizeMock).toBeCalledWith(func, comparer);
+      });
+
+      it('uses shallowComparer by default', () => {
+        createSelector(jest.fn(), jest.fn());
+
+        expect(memoizeMock).toBeCalledWith(expect.anything(), shallowComparer);
+      });
```

Comment:
This test seems to be redundant, as it checks memoize implementation

Reply:
I do not agree with you
This test checks that we pass shallow compare to memorize inside tested function by default.
And we don't check the memoize implementation here because memoize function is mocked :)

Reply:
I see, it's for default parameter value. It's OK, you're right.